.github/workflows/main.yml

```diff
@@ -55,14 +67,6 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
-    - name: Run tests
-      run: |
-        python -m unittest discover tests -v
-    - name: Generate coverage report
-      run: |
-        pip install pytest
-        pip install pytest-cov
-        pytest --cov=./ --cov-report=xml
- 
-    #- name: Upload coverage to Codecov
-          
+    - name: Run tests and generate coverage report
+      run: coverage run -m unittest discover tests -v 
+    #- name: Upload coverage to Codecov          
```

Comment:
Is uploading coverage is remaining intentionally @vj-codes?

Reply:
> Is uploading coverage is remaining intentionally @vj-codes?

Yes